tests/test_third_party_repositories.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2014 Christian Buia
+# Copyright 2025 plyara Maintainers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Test plyara against rule repositories"""
+import requests
+import unittest
+
+from io import BytesIO
+from pathlib import Path
+from tempfile import TemporaryDirectory
+from zipfile import ZipFile
+
+from plyara.core import Plyara
+
+
+class TestPublicYaraRules(unittest.TestCase):
+    """Check parsing of third-party YARA rules"""
+
+    def test_third_party_rules(self):
+        # Perform testing against a set of public YARA rule repositories to assess parsing capability
+        projects = [
+            "AlienVault-Labs/AlienVaultLabs",
```

Comment:
It looks like this repo is the only one causing problems. And this problem is quite large. Can you just comment this line out and leave a note at the end of the comment pointing to the issue where this problem should be solved?